src/aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -563,16 +564,36 @@ def handle_electronic_convergence_not_reached(self, calculation):
 
         Decrease the mixing beta and fully restart from the previous calculation.
         """
-        factor = self.defaults.delta_factor_mixing_beta
+        import numpy
+
+        scf_accuracy = calculation.tools.get_scf_accuracy(0)
```

Comment:
If it is a `relax`, is `index=0` referring to the last scf? Or should it be `-1`?

Reply:
Fair point, I just copied it ^^